includes/api/class-wc-rest-orders-controller.php

```diff
@@ -1656,4 +1678,64 @@ public function get_collection_params() {
 
 		return $params;
 	}
+
+	/**
+	 * Does a permissions check for the proper requested blog
+	 *
+	 * @param WP_REST_Request $request
+	 *
+	 * @return bool $permission
+	 */
+	public function network_orders_permissions_check( $request ) {
+		$blog_id = $request->get_param( 'blog_id' );
+		$blog_id = $blog_id ?: get_current_blog_id();
```

Comment:
Is it correct?
Why check for a $blog_id if always will be the `get_current_blog_id()` ?